src/Agent/NewRelic/Agent/Core/Segments/DatastoreSegmentData.cs

```diff
@@ -233,6 +233,13 @@ public override void SetSpanTypeSpecificAttributes(SpanAttributeValueCollection
             {
                 AttribDefs.DbServerPort.TrySetValue(attribVals, Port.Value);
             }
+
+            // This shouldn't be necessary since we're already setting DbServerAddress above
+            //// For AWS Redshift relationship
+            //if (Host.EndsWith(".redshift.amazonaws.com"))
+            //{
+            //    AttribDefs.ConfigurationEndpointAddress.TrySetValue(attribVals, Host);
+            //}
```

Comment:
Do we still need this?

Reply:
Definitely not, thanks for catching that.